### ISSUE TITLE
Animation.isRunning not exposed

### ISSUE THREAD

**Issue by emilianobovetti** [mdgriffith/elm-style-animation#49]
In my `view` function I need to know if an animation is running to show appropriate elements accordingly. I noticed the function `Animation.isRunning` and I'm wondering if there is a reason why it's not exposed or if I'm using the wrong approach. Thank you.

**Comment by mdgriffith** [comment]
Would using `Animation.Messenger.send` to send a Msg at the end of a step work?

**Comment by emilianobovetti** [comment]
Of course I see there are alternatives, but in that case I should add a field to my application model, send a message and set the field while `Animation.Model.Animation` already stores this information and `Animation.isRunning` reads it, it's just not exposed. Am I missing something?

**Comment by mdgriffith** [comment]
I hear that, but I don't think using `isRunning` directly will lead you down a nice road.  Here's what I'm thinking:

The problem is that then you're saying "have this action occur whenever this animation isn't running".  What if the animation is interrupted halfway through to go to some other target where you don't want that trigger?

I'm guessing here, but likely for your specific situation you've made it so that this animation isn't going to be interrupted.  

Basically using `Animation.Messenger` is more specific and will keep you in a nice place if you need to build on things later.

**Comment by emilianobovetti** [comment]
It sounds reasonable, thank you for prompt reply and for the pointer. Cheers!
